nixos/modules/config/iproute2.nix

```diff
@@ -4,6 +4,17 @@ with lib;
 
 let
   cfg = config.networking.iproute2;
+
+  etc_iproute2 = pkgs.stdenv.mkDerivation {
+    name = "etc-iproute2";
+    src = pkgs.iproute2;
+    buildPhase = ''
+      mkdir $out
+      ln -s $src/etc/iproute2/* $out/
+      rm -f $out/rt_tables
+      cat $src/etc/iproute2/rt_tables ${builtins.toFile cfg.rttablesExtraConfig} > $out/rt_tables
```

Comment:
`builtins.toFile` accepts two arguments, name and contents: <https://nixos.org/manual/nix/stable/language/builtins.html#builtins-toFile>.